### PR TITLE
DataHub namechange

### PR DESCRIPTION
**Pull request by Fernerkundung**
Changing the documentation and comments from `Sentinels Scientific Data Hub (SciHub)` to the new name `Copernicus Open Access Hub`.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-slate
```
Comment (valgur) [inline _config.yml]: Is this commit intentional? It's quite old and not on the PR's branch.

Comment (Fernerkundung) [inline _config.yml]: It is not, and it's not supposed to be in this branch. It was however reverted with the later commit d1584e2. Should I rebase to remove it and create a new PR so it doesnt turn up in the master git logs?

Comment (valgur) [inline _config.yml]: You should try rebasing the current branch on the most recent `origin/master` commit. I'd expect that to fix this.

Comment (valgur) [inline _config.yml]: The changes look good otherwise 👍

**Pull request merged**